File: java/strategy/DuckSimulator/client/RubberDuck.java
package client;

import behavior.fly.FlyNoWay;
import behavior.quack.MuteQuack;

public class RubberDuck extends Duck {

    public RubberDuck(String name) {
        super(name);
        flyBehavior = new FlyNoWay(name);
        quackBehavior = new MuteQuack(name);
    }
}

File: kotlin/strategy/DuckSimulator/client/RubberDuck.kt
package client

import behavior.fly.FlyNoWay
import behavior.quack.MuteQuack

class RubberDuck(duckName: String) : Duck(duckName) {

    init {
        flyBehavior = FlyNoWay(duckName)
        quackBehavior = MuteQuack(duckName)
    }

}